crates/storage_impl/src/payouts/payouts.rs

```diff
@@ -701,6 +725,124 @@ impl<T: DatabaseStore> PayoutsInterface for crate::RouterStore<T> {
         self.filter_payouts_by_constraints(merchant_id, &payout_filters, storage_scheme)
             .await
     }
+
+    #[cfg(feature = "olap")]
+    #[instrument(skip_all)]
+    async fn get_total_count_of_filtered_payouts(
+        &self,
+        merchant_id: &common_utils::id_type::MerchantId,
+        active_payout_ids: &[String],
+        connector: Option<Vec<PayoutConnectors>>,
+        currency: Option<Vec<storage_enums::Currency>>,
+        status: Option<Vec<storage_enums::PayoutStatus>>,
+        payout_type: Option<Vec<storage_enums::PayoutType>>,
+        _storage_scheme: MerchantStorageScheme,
```

Comment:
We can remove `storage_scheme` parameter if it is not being used anywhere, in both the `get_total_count_of_filtered_payouts()` and `filter_active_payout_ids_by_constraints()` methods.